### ISSUE TITLE
Issues with installation using Multi-MC 5 0.6.13

### ISSUE THREAD

**Issue by FinalstarmanDX** [simibubi/Above-and-Beyond#75]
After attempting to install Create Above and Beyond 1.1 manually, from an archive, and by directly downloading the archive through MultiMC the same fatal error loading 2 mods occurs. I have included 5 crash reports from my attempts to launch the instance in question. I attempted tweaking which version of Forge was being used, however, all instances were tested first with Forge 36.2.8 as the installation page suggested. Minecraft will successfully load custom title screen colors and then crash before reaching the title screen producing a "[Render thread/FATAL] [ne.mi.fm.ModLoader/LOADING]: Failed to complete lifecycle event CONSTRUCT, 2 errors found" message in my active log.
[crash-2021-11-07_21.13.19-fml.txt](https://github.com/simibubi/Above-and-Beyond/files/7494269/crash-2021-11-07_21.13.19-fml.txt)
[crash-2021-11-07_21.23.23-fml.txt](https://github.com/simibubi/Above-and-Beyond/files/7494270/crash-2021-11-07_21.23.23-fml.txt)
[crash-2021-11-08_02.51.40-fml.txt](https://github.com/simibubi/Above-and-Beyond/files/7494271/crash-2021-11-08_02.51.40-fml.txt)
[crash-2021-11-07_20.18.14-fml.txt](https://github.com/simibubi/Above-and-Beyond/files/7494272/crash-2021-11-07_20.18.14-fml.txt)
[crash-2021-11-07_20.39.21-fml.txt](https://github.com/simibubi/Above-and-Beyond/files/7494273/crash-2021-11-07_20.39.21-fml.txt)

**Comment by FinalstarmanDX** [comment]
I was able to make a hotfix by deleting CBMultipart and Project red, however, I imagine this is not an actual solution.

**Comment by Geartwo** [comment]
I have the modpack also runing in MultiMC and it works like a charme. What Java Version do you use? If you use anything higher than 11 it will crash.

**Comment by tehbeard** [comment]
Yeah this is the same issue I had with 1.0 as my default java is 16.

Make sure you're on 8.

**Comment by FinalstarmanDX** [comment]
Did some poking and that's definitely the issue. I am in a unique situation where I will have to install multiple versions of Java SDK, so this is very much only an issue for me. Closing as solved.
